tokio-console/src/config.rs

```diff
@@ -528,11 +531,20 @@ impl Config {
             .clone();
         match target_addr.scheme_str() {
             Some("file" | "http" | "https") => {}
+            #[cfg(feature = "vsock")]
+            Some("vsock") => {}
             _ => {
+                #[cfg(feature = "vsock")]
+                return Err(color_eyre::eyre::eyre!(
+                "invalid scheme for target address {:?}, must be one of 'file', 'http', 'https', or 'vsock'",
```

Comment:
Rather than repeating the whole error co traction, we can probably use an `if_cfg!` just to create the list of allowed protocols.